src/descriptions.js

```diff
@@ -15,99 +15,99 @@ module.exports = {
   seer: "Each night, you can select a player to uncover their role by doing `+check <player>`. Your aura is **Good** and you belong to the **Village** team.",
-  analyst: "Each night you can select two players to uncover their alignment: good, evil, or unknown. Evil players belong to the werewolves and good players belong to the villagers team. If both players have the same aura, you will be blocked from checking again the next night. Your aura is **Good** and you belong to the **Village** team.",
-  "aura seer": "Each night, you can select a player to uncover their alignment by doing `+check <player>`: Good, Evil or Unknown. Evil players belong to the werewolves team and Good players belong to the villagers team.\n\nUnknown targets can be: Gunner, Jailer, Marksman, Medium, Witch, Forger, Beast Hunter, Fortune Teller, Alpha Werewolf, Fool, Headhunter, Serial Killer, Arsonist, Bomber, Corruptor, Cannibal, Illusionist, Bandit, Accomplice, Sect Leader, Zombie, Sect Hunter, Mad Scientist, Naughty Boy, Doppelganger or Lone Wolf.\n\nYour aura is **Good** and you belong to the **Village** team.",
-  "spirit seer": "Each night, you select up to two players by doing `+check <player1> [player2]` (you can check 1 person too). At the beginning of the next day you will be informed if either of those two players have killed last night, or during the previous day in case they are a solo killer. Your aura is **Good** and you belong to the **Village** team.",
-  mortician: "Each night you can select a player killed by non-villagers and uncover two possible suspects who might have killed that player. If the player was killed by a solo killer, you will uncover three suspects instead. Your aura is **Good** and you belong to the **Village** team.",
+  analyst: "Each night you can select two players to uncover their alignment by doing `+check <player 1> <player 2>`: good, evil, or unknown. Evil players belong to the werewolves and good players belong to the villagers team. If both players have the same aura, you will be blocked from checking again the next night. Your aura is **Good** and you belong to the **Village** team.",
+  "aura seer": "Each night, you can select a player to uncover their alignment by doing `+check <player>`: Good, Evil or Unknown. Evil players belong to the werewolves team and Good players belong to the villagers team. Your aura is **Good** and you belong to the **Village** team.",
+  "spirit seer": "Each night, you select __up to two players__ by doing `+check <player 1> (player2)`. At the beginning of the next day you will be informed if either of those two players have killed last night, or during the previous day in case they are a solo killer. Your aura is **Good** and you belong to the **Village** team.",
```

Comment:
```suggestion
  "spirit seer": "Each night, you select __up to two players__ by doing `+check <player 1> [player2]`. At the beginning of the next day you will be informed if either of those two players have killed last night, or during the previous day in case they are a solo killer. Your aura is **Good** and you belong to the **Village** team.",
```

[ ] = optional
( ) = sub command (like +xp (add) )